--- a/anz.casclient-1.0.1-py2.7.egg/anz/casclient/validationspecification.py	2014-12-11 14:23:57.308906055 +0200
+++ b/anz.casclient-1.0.1-py2.7.egg/anz/casclient/validationspecification.py	2014-12-11 14:26:52.205972387 +0200
@@ -159,7 +159,7 @@
             
             return Assertion( principal )
         except Exception, e:
-            raise InternalException( str(e) )
+            raise ValueError( str(e) )
 
 class Cas20ProxyTicketValidator( Cas20ServiceTicketValidator ):
     ''' Extension Service Ticket validation to validate service tickets and
