--- a/Products.PluggableAuthService-1.10.0-py2.7.egg/Products/PluggableAuthService/PluggableAuthService.py	2015-02-19 13:33:05.831739815 +0200
+++ b/Products.PluggableAuthService-1.10.0-py2.7.egg/Products/PluggableAuthService/PluggableAuthService.py	2015-02-19 13:33:45.329932042 +0200
@@ -603,11 +603,9 @@
 
             try:
                 credentials = extractor.extractCredentials( request )
-            except _SWALLOWABLE_PLUGIN_EXCEPTIONS:
+            except _SWALLOWABLE_PLUGIN_EXCEPTIONS as e:
                 reraise(extractor)
-                logger.debug( 'ExtractionPlugin %s error' % extractor_id
-                            , exc_info=True
-                            )
+                logger.warning( 'ExtractionPlugin %s failed (%s)' % (extractor_id, str(e)))
                 continue
 
             if credentials:
